osu.Game/Online/OnlineViewContainer.cs

```diff
@@ -18,7 +18,7 @@ public class OnlineViewContainer : Container, IOnlineComponent
     {
         private readonly Container placeholderContainer;
         private readonly Placeholder placeholder;
-        private readonly LoadingAnimation loading;
+        internal readonly LoadingAnimation LoadingAnimation;
```

Comment:
Doesn't need to be that exposed, what you're looking for is making this protected and expose it on test sub-class with `public new LoadingAnimation => base.LoadingAnimation`.